src/command/command.js

```diff
@@ -90,6 +90,24 @@ yargs
             wordpressStart(argv.flotiqApiKey, argv.wordpressUrl, yargs.argv['json-output']);
         }
     })
+    .command('contentful-import [flotiqApiKey] [contentfulSpaceId] [contentfulContentManagementToken] [translation]', 'Import Contentful to Flotiq', (yargs) => {
```

Comment:
parameter [flotiqApiKey] is optional (can be read from ENV) move this to end, before [translation]